doc/classes/Image.xml

```diff
@@ -166,52 +175,56 @@
 			<argument index="0" name="color" type="Color">
 			</argument>
 			<description>
-				Fills an [code]Image[/code] with a specified [Color]
+				Fills this [code]Image[/code] with a specified [Color].
```

Comment:
Why the [code] tags around "Image" for only this description?

"Fills the image with a given [Color]."